plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/math/KVector.java

```diff
@@ -175,6 +175,18 @@ public KVector reset() {
         this.y = 0.0;
         return this;
     }
+    
+    /**
+     * Resets this vector to the value of the other vector.
+     * 
+     * @param other the vector whose values to copy.
+     * @return {@code this}
+     */
+    public KVector reset(final KVector other) {
```

Comment:
True. I won't bore you with how this unfortunate piece of naming crept into the code... :D